lib/requireRule.js

```diff
@@ -3,10 +3,15 @@
 const importLazy = require("import-lazy")(require);
 const rules = require("./rules");
 
-module.exports = function(ruleName) {
+function requireRuleUnsafe(ruleName) {
```

Comment:
What is the reason to make this function?

Reply:
After looking how currently we load lazily rules in our public API I found out that there is unnecessary check [here](https://github.com/stylelint/stylelint/blob/master/lib/index.js#L19), so we can omit it using `unsafe` require. Theoretically this change reduce code complexity by `O(n * x)`, where `n` is amount of rules and `x` is complexity to check entry in `Set`. We don't change `Set` between comparisons and amount of rules is not much(likelihood of collisions is very small), so in real world I think one check takes not much times (this change was the least valued I think)

Reply:
Thank you!

I think we need to rename unsafe for something less scary. Usually “unsafe” things are experimental or fragile, which is not the case for us.

I can't come up with a better name. Which makes me think. Is change to requireRule has real benefits. It make code harder to understand. 1-2ms improvement not worth to make code more complex.